concent_api/core/tests/test_unit_transfer_operation.py

```diff
@@ -109,27 +110,27 @@ def test_that_file_transfer_token_for_concent_is_never_out_of_date(self):
         )
         self.assertTrue(token.timestamp < token.token_expiration_deadline)
 
-    def test_that_download_file_transfer_token_for_golem_client_is_can_be_out_of_date(self):
+    def test_that_download_file_transfer_token_for_golem_client_cant_be_out_of_date(self):
```

Comment:
this changes the bahaviuor (so it is no refactoring anymore) and must be clarified

Reply:
Clarified with @cameel, I've revert this to previous functionality.